recipe/bld.bat

```diff
@@ -0,0 +1,3 @@
+copy /y %RECIPE_DIR%\isl_declaration_macros_expanded.h .
```

Comment:
Header file shipped with islpy is for isl development version. Since the conda package `isl` packages 0.18, I generated this file by using it and removing the last line `ISL_DECLARE_MULTI_CMP(aff)`

Reply:
OIC, thanks!